sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/statsbeat/ResourceProvider.java

```diff
@@ -8,6 +8,7 @@ enum ResourceProvider {
     RP_APPSVC("appsvc"),
     RP_VM("vm"),
     RP_AKS("aks"),
+    RP_SPRING_CLOUD("springcloud"),
```

Comment:
Is it [Azure Spring apps](https://azure.microsoft.com/en-us/products/spring-apps)?

Reply:
yeah, considered that, but our marker file uses springcloud.. keep it consistent.